### DISCUSSION TITLE
Resigned WebView APK signing keys

### DISCUSSION THREAD

**Discussion by kosmosnautti** [seaky/nspanel_pro_tools_apk#45] [Q&A]
I have a similar device with bigger screen from the same manufacturer as NSPanel Pro. I was able to update the outdated WebView on that with the resigned WebView 108 APK found on the documentation, first of all thanks for that!

I was just curious if it's possible to share the keys which were used to sign that APK, so that I could update it even further, say to version 120, or if @seaky can provide a resigned APK for a later version?

EDIT:
Found the keys, here they are in case anyone else is looking for them to do some signing on their own: 
[signing_keys.zip](https://github.com/seaky/nspanel_pro_tools_apk/files/14025519/signing_keys.zip)

Passphrase for the ready to use .jks-keystore is "android".

**Comment by seaky** [comment]
sorry I was busy with v2.0.1 but as I see you've found the solution!

btw what is your experince with the big borther? 
And is my app working on it?

thanks  in advance,
Robi

**Comment by kosmosnautti** [comment]
I've had limited time to test the device, but I must say that I do like the 10" screen over the smaller 4" ones. Haven't tested if this app works on it, at least not yet.

The fact that it comes rooted from the factory and with a plain Android 11 is also a plus.

**Reply by ianfretwell** [reply]
Could you actually provide a link to this product because there's absolutely nothing on the Sonoff website any bigger than the NSPanelPro....thx...

**Reply by kosmosnautti** [reply]
It's not from Sonoff, but from the original device manufacturer which products Sonoff (as far as I know) uses under their brand. This is the one I'm talking about: https://www.smatek.com/index.php?m=content&c=index&a=show&catid=11&id=38&page=1

There are two variants of this device available, one with Tuya junk preinstalled and another one with just a plain rooted Android 11. Same variants are also available for other Android-devices that they have (with a different version number of Android).
